docs/DeepSeek-R1-0528-int4-asym-awq-inc.md

```diff
@@ -7,7 +7,7 @@ base_model:
 
 ## Model Details
 
-This model is an int4 model with group_size  64 and symmetric quantization of [deepseek-ai/DeepSeek-R1-0528](https://huggingface.co/deepseek-ai/DeepSeek-R1-0528) generated by [intel/auto-round](https://github.com/intel/auto-round) algorithm.  Some layers are fallback to 4/16 bits. Refer to  Section "Generate the model" for more details of mixed bits setting.
+This model is an int4 model with group_size  64 and asymmetric quantization of [deepseek-ai/DeepSeek-R1-0528](https://huggingface.co/deepseek-ai/DeepSeek-R1-0528) generated by [intel/auto-round](https://github.com/intel/auto-round) algorithm.  
 
```

Comment:
delete this in no-mixed recipes “this model could NOT run on other severing frameworks.”